client-course-schedulizer/src/utilities/helpers/readCSV.ts

```diff
@@ -132,6 +126,21 @@ export const csvStringToSchedule = (csvString: string): Schedule => {
 };
 
 export const insertSectionCourse = (schedule: Schedule, section: Section, course: Course) => {
+  const { meetings } = section;
+
+  // Check if any meetings are empty, and should be removed
+  let i = 0;
+  let numRemoved = 0;
+  const meetingsCopy = meetings;
+  meetings.forEach((meeting) => {
```

Comment:
You can use the `forEach()` index parameter here too

Reply:
325d1a9 and 172cce3